src/client/components/LocationComponent/LocationComponent.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+class LocationComponent extends React.Component {
+  state = { adress: '' };
+
+  componentDidMount() {
+    fetch(
+      'https://api.opencagedata.com/geocode/v1/json?q=55.661869+12.540742&key=216a941e5d7e4b5dbd4c778c87486d35',
+    )
+      .then((respoonse) => respoonse.json())
```

Comment:
This doesn't affect the code, but here I think "response" could be better than "resp**oo**nse".